_includes/setup/next-steps-win-linux.md

```diff
@@ -0,0 +1,5 @@
+## Next steps
+
+Now, follow our [Getting Started guide](/getting-started/)
+to run your first Flutter app on Android. To develop for iOS in addition to Android,
+follow the [Flutter Setup for Mac](/setup-mac/) instructions.
```

Comment:
Pointing to Flutter on Mac in the `next-step-win-linux.md` file seems strange. Do we expect Windows/Linux users to also have a mac machine?